python/src/nanoarrow/_lib.pyx

```diff
@@ -861,6 +867,27 @@ cdef class CSchemaView:
         return _repr_utils.schema_view_repr(self)
 
 
+cdef class CLayout:
+    cdef ArrowLayout* _layout
+    cdef object _base
+
+    def __cinit__(self, base, uintptr_t ptr):
+        self._base = base
+        self._layout = <ArrowLayout*>ptr
+
+    @property
+    def buffer_data_type_id(self):
+        return tuple(self._layout.buffer_data_type[i] for i in range(3))
```

Comment:
Can we define the magic number 3 as a constant or variable somewhere? (or better yet, is it possible to take the `len()` of some object?)